src/Npgsql/PublicAPI.Shipped.txt

```diff
@@ -990,6 +990,7 @@ Npgsql.Replication.PhysicalReplicationConnection.PhysicalReplicationConnection(s
 Npgsql.Replication.PhysicalReplicationConnection.StartReplication(Npgsql.Replication.PhysicalReplicationSlot? slot, NpgsqlTypes.NpgsqlLogSequenceNumber walLocation, System.Threading.CancellationToken cancellationToken, uint timeline = 0) -> System.Collections.Generic.IAsyncEnumerable<Npgsql.Replication.XLogDataMessage!>!
 Npgsql.Replication.PhysicalReplicationConnection.StartReplication(NpgsqlTypes.NpgsqlLogSequenceNumber walLocation, System.Threading.CancellationToken cancellationToken, uint timeline = 0) -> System.Collections.Generic.IAsyncEnumerable<Npgsql.Replication.XLogDataMessage!>!
 Npgsql.Replication.PhysicalReplicationSlot
+Npgsql.Replication.PhysicalReplicationSlot.PhysicalReplicationSlot(string! slotName) -> void
```

Comment:
Note this for the new public API analyzers.

Since we're making this public API change in a patch release (should not normally happen...), I'm including this in the "shipped" file rather than the "unshipped". In the regular case, changes would only be released for 6.0.0 (or vNext), and would go into "unshipped" instead.